trino/trino.go

```diff
@@ -909,6 +921,14 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 				continue
 			}
 
+			if arg.Name == accessTokenConfig {
```

Comment:
We already have support to copy args prefixed with `trinoHeaderPrefix` into headers. Why is a separate option needed for copying the access token?

Reply:
Hi, 
all args prefixed with `trinoHeaderPrefixed` are copied into the header with the same name.
Here we want to be able to authenticate on Trino with an OAuth JWT forwarded by a tier application (e.g. `grafana-trino` plugin). Hence, we need to have the header name `Authorization`, which is impossible with the `trinoHeaderPrefixed`.